Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/RequestsManager.java

```diff
@@ -0,0 +1,49 @@
+package edu.asu.conceptpower.app.core.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.db4o.ObjectSet;
+
+import edu.asu.conceptpower.app.core.IRequestsManager;
+import edu.asu.conceptpower.app.db4o.IRequestsDBManager;
+import edu.asu.conceptpower.core.ReviewRequest;
+
+@Service
+public  class RequestsManager implements IRequestsManager{
+
+  
+    @Autowired
+    private IRequestsDBManager dbClient;
+   
+    
+    
+    /* (non-Javadoc)
+     * @see edu.asu.conceptpower.app.core.IRequestsManager#addReviewRequest(edu.asu.conceptpower.core.ReviewRequest)
+     */
+    public void addReviewRequest(ReviewRequest newReviewRequest) {
+        
+        dbClient.store(newReviewRequest);
+    }
+    
+    /* (non-Javadoc)
+     * @see edu.asu.conceptpower.app.core.IRequestsManager#getReview(java.lang.String)
+     */
+    public ReviewRequest getReview(String conceptId) {
+        
+        ReviewRequest exampleEntry = new ReviewRequest();
+        exampleEntry.setConceptId(conceptId);
+        
+        ObjectSet<ReviewRequest> results = dbClient.getReviewRequestForConcept(exampleEntry);
```

Comment:
you're still giving it an example object, you need to pass a string so you don't have to change this method when we replace the db backend